### ISSUE TITLE
Fix the marketplace bug!

### ISSUE THREAD

**Issue by ironboy36** [minecraft-linux/mcpelauncher-manifest#562]
I pressed on the market place and then it is stuck loading!
![Screen Shot 2021-07-09 at 3 07 40 PM](https://user-images.githubusercontent.com/77495097/125125545-bbdf0100-e0c7-11eb-9f18-6e4fe2dee56f.png)
I do have a good internet connection (32 MB/S)

**Comment by ironboy36** [comment]
Info:

Computer: MacBook (13-inch, Early 2009)
Minecraft version: Minecraft Bedrock 1.16.221
![Screen Shot 2021-07-09 at 4 09 57 PM](https://user-images.githubusercontent.com/77495097/125131073-26943a80-e0d0-11eb-84b5-54d87e4eb9cc.png)

**Comment by abourget** [comment]
Yes, I submitted more details in this bug report: https://github.com/minecraft-linux/mcpelauncher-manifest/issues/509

I'd add "please". Is there a donation page to help encourage this fix?
